doc/api/next_api_changes/2019-05-21-CB.rst

```diff
@@ -0,0 +1,12 @@
+`matplotlib.axes.Axes.indicate_inset` returns a 4-tuple as documented
+---------------------------------------------------------------------
+
+In <= 3.1.0, `matplotlib.axes.Axes.indicate_inset` and
+`matplotlib.axes.Axes.indicate_inset_zoom` were documented as returning
+a 4-tuple of `matplotlib.patches.ConnectionPatch`es, where in fact they
+returned a 4-length list.
+
```

Comment:
I guess I'm not seeing the difference here.  What is the advantage of the change below?  If its just because we don't think a list is a tuple, then I'd argue just changing the docs to say it is a list. But practically, what is the difference?

Reply:
I'm not sure I understand. Lists and tuples are different. They do different things. They have different methods. At the very least, the documentation should match the implementation.

If you tell someone you're giving them a duck and then show up with a cat, they're going to be annoyed that they've spent some time digging a pond.

> I'd argue just changing the docs to say it is a list

This is what I did initially. However, in this case, the return value has a fixed length and the indices are meaningful, so an optional tuple is closer to the meaning of what's being represented. I suggested the change in an earlier comment and said my only reservation was about it being a breaking API change, and you responded that breaking changes were fair game as it's an experimental feature.

Reply:
Ok I was looking at the wrong change.  In general it’s better not to have too many style changes unless they help pass PEP8 with fewer exceptions.  

I’m not clear if a tuple or list is particularly better here.  I guess the fact that it’s always a four tuple would argue for that.  My apologies that my previous comment was taken to mean I thought it *should* be changed, just that it *could* if there is a good reason.

Reply:
Returning a tuple makes sense semantically.